change/office-ui-fabric-react-2020-05-11-12-07-43-master.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Clicking DocumentCard always opens the link in the same page. I have a requirement where I prefer to load it in a new tab.",
```

This message is used in the changelog, so it should be something more like this:
```suggestion
  "comment": "DocumentCard: add prop to open link in a new tab.",
```